markdown/blog/2025-gsoc-phase-1.md

```diff
@@ -0,0 +1,179 @@
+---
+title: "2025 Google Summer of Code: Phase 1"
+date: 2025-05-27T06:00:00+01:00
+type: Community
+tags:
+  - Mentorship
+  - GSoC
+  - AsyncAPI
+  - OSS
+cover: /img/posts/gsoc-25.webp
+authors:
+  - name: Azeez Elegbede
+    photo: /img/avatars/ace.webp
+    link: https://www.linkedin.com/in/aelegbede/
+    byline: AsyncAPI Preacher 
+---
+
+**At AsyncAPI, Summer means one thing: Google Summer of Code**
+
+Every year, we eagerly anticipate this season, not just because of the program itself, but because of what it represents for our community. GSoC isn't just about the stipend contributors receive for their participation. For us, it's about the doors it opens, the chance to welcome new developers, foster fresh perspectives, and nurture contributors who often go on to become long-term contributors of the project. 
+
+Participating in Google Summer of Code is an opportunity to invest in mentorship, give back to the AsyncAPI community, and continue building the future of event-driven APIs, one contributor at a time. 
+
+## How We Prepared
+
+One of the most challenging aspects of participating in Google Summer of Code is getting accepted as a mentoring organization. At AsyncAPI, we knew that increasing our chances of being selected would require early and deliberate planning, so we started preparing well before Google even announced the official program dates.
+
+Here’s how we laid the groundwork for a successful GSoC 2025 first phase:
+
+- **Kicking Off with Community Discussion:**
+  We initiated an open [GitHub Discussion](https://github.com/orgs/asyncapi/discussions/1633) to serve as the central hub for all things GSoC-related. This thread became our base for collaboration and coordination throughout the application process. It helped us engage the community early, keep everything transparent, and foster involvement from contributors and mentors.
+
+- **Collecting Project Ideas:**
+  Identifying meaningful and impactful projects is a key part of the GSoC application. Through the GitHub Discussion, we invited community members to propose project ideas they were excited about. This resulted in 15 incredible suggestions, including 9 solid proposals that had the potential to evolve into full-fledged GSoC projects.
+
+- **Calling for Mentors:**
+  Mentors are the heart of a successful GSoC experience. Rather than manage a separate process, we used the same discussion thread to invite experienced contributors and maintainers to step forward as potential mentors. This approach helped us keep all conversations in one place and encouraged mentors to engage with idea proposals, provide feedback, and shape the project scope collaboratively.
```

Comment:
```suggestion
  Mentors are the heart of a successful GSoC experience. Rather than manage a separate process, we used the same discussion thread to invite experienced contributors and maintainers to step forward as potential mentors. This approach helped us keep all conversations in one place and encouraged mentors to engage with idea proposals, provide feedback, and collaboratively shape the project scope.
```